anthropic-bedrock-model-provider/main.py

```diff
@@ -0,0 +1,85 @@
+import json
+import os
+import sys
+
+import boto3
+import claude3_provider_common
+from anthropic import AsyncAnthropicBedrock
+from fastapi import FastAPI, Request
+from fastapi.responses import JSONResponse, StreamingResponse
+
+debug = os.environ.get("GPTSCRIPT_DEBUG", "false") == "true"
+def log(*args):
+    if debug:
+        print(*args)
+
+
+os.environ["AWS_ACCESS_KEY_ID"] = os.environ.get("OBOT_ANTHROPIC_BEDROCK_MODEL_PROVIDER_ACCESS_KEY_ID")
+os.environ["AWS_SECRET_ACCESS_KEY"] = os.environ.get("OBOT_ANTHROPIC_BEDROCK_MODEL_PROVIDER_SECRET_ACCESS_KEY")
+os.environ["AWS_SESSION_TOKEN"] = os.environ.get("OBOT_ANTHROPIC_BEDROCK_MODEL_PROVIDER_SESSION_TOKEN")
+os.environ["AWS_REGION"] = os.environ["AWS_DEFAULT_REGION"] = os.environ.get("OBOT_ANTHROPIC_BEDROCK_MODEL_PROVIDER_REGION")
+
+# Check if any is empty
+if not all([os.environ["AWS_ACCESS_KEY_ID"], os.environ["AWS_SECRET_ACCESS_KEY"], os.environ["AWS_DEFAULT_REGION"], os.environ["AWS_SESSION_TOKEN"]]):
+    print("Please set AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY, AWS_DEFAULT_REGION, AWS_SESSION_TOKEN", file=sys.stderr)
```

Comment:
I think this should print the `OBOT` env vars that it reads these from.

Reply:
changed